apps/music/js/db.js

```diff
@@ -39,9 +40,14 @@ function initDB() {
   });
 
   function metadataParserWrapper(file, onsuccess, onerror) {
-    LazyLoader.load('js/metadata_scripts.js', function() {
-      AudioMetadata.parse(file).then(onsuccess, onerror);
-    });
+    LazyLoader.load(
+      ['/js/metadata_scripts.js', '/js/metadata/album_art.js'],
+      function() {
+        AudioMetadata.parse(file).then(function(metadata) {
+          AlbumArt.process(file, metadata);
+        }).then(onsuccess, onerror);
```

Comment:
That's a mistake, but it might actually be a good idea. It could make parsing a bit more parallelizable, but we'd need more changes elsewhere to allow for that. Specifically, the checks for external album art would still need to be serial, since they ultimately apply to multiple songs.